t/fake-isotovideo.pl

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/perl
+
+if ($ARGV[0] ne '--version') {
+    print("This script is only meant to test the isotovideo version check.\n");
+    exit(-1);
+}
+
+print("Current version is 4.5.1559738889.52a75c17 [interface v15]\n");
```

Comment:
Could you rather fake this within the test code by mocking a calling function to return this string?

Reply:
I can not test the code for invoking `isotovideo --version` by mocking it. I could reduce the test to test the pattern matching only. But for that I need to refactor the engine first to decouple this. However, I thought this PR is big enough so I only changed what was necessary in the engine so far. Therefore I'd like to save this for another PR.